File: client/src/app/shared/models/approvalAuthority.ts

export interface IApprovalAuthority {
    id: number;
    compId: number;
    deptId: number;
    approvalAuthorityName: string;
    priority: number;
    remarks: string; 
    status: string; 
    setOn: Date;
}
 
export class ApprovalAuthority implements IApprovalAuthority {
    id: number=0;
    compId: number;
    deptId: number;
    approvalAuthorityName: string;
    remarks: string; 
    priority: number=null;
    status: string=null; 
    setOn: Date;
}

File: Core/Entities/ApprovalAuthority.cs

using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace Core.Entities

{
    public class ApprovalAuthority:BaseEntity
    {
        public string ApprovalAuthorityName { get; set; }
        public string Remarks { get; set; }
        public string Status { get; set; }
        public int Priority { get; set; }

    }
}